### DISCUSSION TITLE
Has anyone seen or solved the d3d11-compatible gpu problem?

### DISCUSSION THREAD

**Discussion by dpa456** [utmapp/UTM#5058] [UTM Help (macOS)] [answered]
When you try to play a particular game, you get the error d3d11-compatible gpu (feature level 11.0, shader model 5.0) is required to run the engine.
you know that this error is directX, cpp++ not install or gpu not support.
Most of the time, I installed the latest version, but it didn't work well.
Do you think it's a scope that doesn't apply? 

I've seen the reference.
https://github.com/utmapp/UTM/issues/161

**Comment by osy** [comment, marked as answer]
Gaming/GPU is not supported by UTM currently.

**Reply by dpa456** [reply]
thank you for replay, :)

**Reply by martin-braun** [reply]
@osy Thank you so much for UTM. It has the potential to beat Parallels, the lack of graphics acceleration on Windows is just the last piece that would make it beat as far as I know. If I got it right, acceleration is only possible for linux VMs by using qemu-virgl. Is someone working on bringing graphics acceleration to Windows VMs as well?

**Reply by tifasoftware** [reply]
I believe we are aiming to bring soon bring virgl to UTM. I gave a suggestion to bring in emulation for the old Voodoo cards for classic 3d games like (Unreal Gold, Deus EX, etc.), but we decided to focus on just virgl. 

So in the meantime, you have to pay for parallels right now, like I am. Also, i believe Asahi is finding some success in running some proton games on M1, but I feel there may be some catches.

**Reply by martin-braun** [reply]
@tifasoftware Thanks for sharing your knowledge about the current status. I'd like to emphasize that a solution for M1, M2, etc. and <b>T2</b> Macs would be desirable. I still own an Intel-based Mac and would enjoy to see the break of Parallels market dominance for Intel machines as well.

**Comment by shell1986** [comment]
Hello, thanks for creating this thread. I would like to occasionally read the news about how the development and progress towards the GPU is going.
Maybe the moderator should create a fixed topic where all these questions will flock. I would gladly read it silently from time to time.

**Reply by osy** [reply]
If there’s any progress of GPU work, I promise it will be widely broadcasted.
